includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -937,6 +937,7 @@ protected function get_wizard_shipping_methods( $country_code, $currency_code )
 						'default_value' => __( 'Cost', 'woocommerce' ),
 						'description'   => __( 'What would you like to charge for flat rate shipping?', 'woocommerce' ),
 						'required'      => true,
+						'pattern'       => '^[+-]?[0-9]{1,3}(?:[0-9]*(?:[.,][0-9]{2})?|(?:,[0-9]{3})*(?:\.[0-9]{2})?|(?:\.[0-9]{3})*(?:,[0-9]{2})?)$',
```

Comment:
Due to the way browsers display pattern errors, I don't think we should add this constraint. Having a server-side error thrown for invalid formats should be sufficient.

Reply:
Hey @TimBHowe,

Did you want to wrap this PR up?

Reply:
Ok, I'll remove it.